src/inc/eventtracebase.h

```diff
@@ -218,14 +222,25 @@ struct ProfilingScanContext;
 extern UINT32 g_nClrInstanceId;
 
 #define GetClrInstanceId()  (static_cast<UINT16>(g_nClrInstanceId))
+#if defined(FEATURE_PAL) && (defined(FEATURE_EVENT_TRACE) || defined(FEATURE_EVENTSOURCE_XPLAT))
+#define KEYWORDZERO 0x0
+
+/***************************************/
+/* Tracing levels supported by CLR ETW */
+/***************************************/
+#define ETWMAX_TRACE_LEVEL 6        // Maximum Number of Trace Levels supported
+#define TRACE_LEVEL_NONE        0   // Tracing is not on
```

Comment:
Can we use [EventPipeEventLevel][https://github.com/dotnet/coreclr/blob/master/src/vm/eventpipecommontypes.h#L12] here instead?